### ISSUE TITLE
setZOrderOnTop(true);

### ISSUE THREAD

**Issue by mahmoud-elnaggar** [winsontan520/Android-WScratchView#16]
What if I don't want to make it top of all layouts 
if I write this setZOrderOnTop(true); it doesn't work 
thanks for sharing this lib

**Comment by winsontan520** [comment]
Yes this is the drawback of this library, right now I have no idea to improve it right now
